cpp/ql/src/semmle/code/cpp/exprs/BuiltInOperations.qll

```diff
@@ -387,7 +676,17 @@ class BuiltInOperationHasTrivialMoveConstructor extends BuiltInOperation, @hastr
 }
 
 /**
- * The `__has_trivial_move_assign` type trait.
+ * The `__has_trivial_move_assign` built-in function (used by some
+ * implementations of the `<type_traits>` header).
+ *
+ * Returns if the move-assign operator `C::operator =(C &&c)` is trivial
+ * (i.e., can be generated by the compiler).
+ * ```
+ * template<typename T>
+ *   struct is_trivially_assignable
+ *   : public integral_constant<bool, __is_trivially_assignable(T) >
+ *   { };
```

Comment:
This example is the `__is_trivially_assignable` one but should be `__has_trivial_move_assign`.